src/currency.js

```diff
@@ -0,0 +1,4 @@
+module.exports = function formatAsMoney(dollars) {
+  if ( typeof dollars !== 'number' ) throw new Error( 'expected ('+JSON.stringify(dollars)+') to be a number')
+  return '$'+dollars.toFixed(2)
+}
```

Comment:
I would clean this up a little with ES6, and write clearer code with whitespace and blocks:
```es6
const formatAsMoney = dollars => {
  if( typeof dollars !== 'number' ) {
    throw new Error( `Expected ${JSON.stringify( dollars )} to be a number` )
  }

  return `$${dollars.toFixed( 2 )}`
}

export default formatAsMoney
```